web/templates/edited_notice.hbs

```diff
@@ -1,14 +1,17 @@
+<div id="editing_notifications" class="edit_notifications {{#unless is_typing}}hide{{/unless}}">
+    {{> editing_notifications}}
+</div>
```

Comment:
So I don't think we want to have this HTML in the DOM for every one of hundreds of message elements in the DOM, regardless of whether it's actually being edited. See 
https://chat.zulip.org/#narrow/channel/6-frontend/topic/narrowing.20performance/near/1904245 for some context, but I think we want this to be something that gets inserted if needed and removed when not needed, rather than always present and doing show/hide.